charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -134,32 +132,71 @@ data:
             # This function is responsible for two things. It assigns labels to disambiguate
             # between KPO and K8s executor pods. Also, it handles writing the entrypoint for
             # K8s executor pods. Why we do this second part, you can read below.
-            def pod_mutation_hook(pod: Pod):
+            from airflow.configuration import conf
+            from airflow.version import version
+            def pod_mutation_hook_deprecated(pod):
+              from airflow.contrib.kubernetes.pod import Pod
               extra_labels = {
-                  "kubernetes-executor": "False",
                   "kubernetes-pod-operator": "False"
               }
-              if 'airflow-worker' in pod.labels.keys() or \
+              if 'airflow-worker' in pod.labels.keys() and \
                       conf.get('core', 'EXECUTOR') == "KubernetesExecutor":
-                  extra_labels["kubernetes-executor"] = "True"
+                  # extra_labels["kubernetes-executor"] = "True"
```

Comment:
Why did you comment this out?

Reply:
This is commented out in cloud.

Reply:
Yup this is the configmap in use in cloud at the moment minus some pod label affinity items

Reply:
Ok sounds good